web/explorer/src/components/NavBar.vue

```diff
@@ -8,10 +8,15 @@ import Menubar from "primevue/menubar";
             <div class="flex align-items-center gap-3">
                 <a
                     v-ripple
-                    href="https://github.com/mandiant/capa"
-                    class="flex align-items-center justify-content-center text-color w-2rem"
+                    v-tooltip.right="'Download capa Explorer Web for offline usage'"
+                    href="/capa-explorer-web.zip"
```

Comment:
does vite fix up this path or does it need to be made relative?

Reply:
The href `/capa-explorer-web.zip` is relative to the root of the project (i.e. `web/explorer`), thus I believe it is already a relative path.